.editorconfig

```diff
@@ -0,0 +1,15 @@
+; This file is for unifying the coding style for different editors and IDEs.
+; More information at http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{html,md,js,json,xml}]
```

Comment:
```suggestion
[*.{html,md,js,mjs,json,xml}]
```

Reply:
Is it intentional that CSS is indented differently from JS?

Reply:
No, I just took the `.editorconfig` that was already present without alterations.